### PR TITLE
Object support ES5 compatible

### PR DESCRIPTION
**Pull request by andreafalzetti**

### CODE REVIEW

**Comment by gabrielperales** [comment]
@andreafalzetti You should try to add some tests for objects :)

**Comment by andreafalzetti** [comment]
@gabrielperales I added the tests in a separate pull-request

File: index.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 
-  for (key in context) {
-    find = '\\$\\{\\s*' + key + '\\s*\\}';
-    re = new RegExp(find, 'g');
-    string = string.replace(re, context[key]);
+function _recursive_rendering(string, context, stack) {
+  for (var key in context) {
+    if (context.hasOwnProperty(key)) {
+      if(typeof context[key] === "object") {
+        string = _recursive_rendering(string, context[key], (stack?`${stack}.`:'')+ key);
```
Comment (gabrielperales) [inline index.js:9]: template literals are not compatible with ES5. I think we can keep this library simple and avoid use babel to transpile code to do it ES5 compatible.

**Comment by andreafalzetti** [comment]
that's a good point, I will delete this and submit a new pull request. Thanks @gabrielperales

**Pull request closed without merging**